src/resources/postcss/tickets-commerce/admin/orders/table.pcss

```diff
@@ -104,35 +105,48 @@ body.wp-admin.post-type-tec_tc_order {
 	}
 }
 
-.tribe-external-link {
+.tribe-dashicons {
 	&--code {
 		font-family: monospace;
 	}
 
-	&:after {
-		content: url("../../../../images/tribe-external-link.svg");
+	.dashicons {
 		height: 16px;
 		padding-left: var(--tec-spacer-0);
 		width: 16px;
 	}
+
+	.tribe-icon-trash {
+		background-image: url("../../../../images/admin/trash.svg");
+		background-repeat: no-repeat;
+		background-size: contain;
+	}
 }
 
 .tribe-tickets-commerce-order-status {
+	align-items: center;
 	background: #c3c4c7;
 	border: 1px solid rgba(0, 0, 0, 0.05);
 	border-radius: var(--tec-border-radius-default);
 	color: #2c3338;
 	cursor: inherit !important;
 	display: inline-flex;
+	flex-flow: row nowrap;
+	justify-content: center;
 	line-height: 2.5em;
 	margin: calc(-1 * var(--tec-spacer-0)) 0;
 	max-width: 100%;
 	white-space: nowrap;
 
 	> span {
-		margin: 0 1em;
+		margin-right: 14px;
 		overflow: hidden;
 		text-overflow: ellipsis;
+
+		&:first-child {
+			margin-left: 14px;
+			margin-right: 4px;
```

Comment:
-tec-spacer-0

I'd check with the designer too about changing these to fit our custom props - we have them to help enforce consistency.